src/db/functions.ts

```diff
@@ -1,32 +0,0 @@
-// WARNING: We haven't set up a shared library for these types, so these must be synchronized between
```

Comment:
I believe this comment is obsolete after this PR. Delete if so.

Reply:
This file is being deleted! But good catch, before my last commit I hadn't deleted the comment in the shared types file. Now it is gone :)

https://github.com/wordplaydev/wordplay/pull/694/files#diff-a76a2413f6b49c49a2f7ada05258ff52b91ca337f5b8c95b2c6c2b4d878096bdL1-L2